content/03-type-system.md

```diff
@@ -562,19 +562,19 @@ Given an identifier `i`, the algorithm is as follows:
 8. If a static named `i` is explicitly imported, resolve to it and halt.
 9. If `i` starts with a lower-case character, go to 11.
 10. If a type named `i` is available, resolve to it and halt.
-11. If the expression is not in untyped mode, go to 14
+11. If the expression is not in untyped mode, go to 14.
 12. If `i` equals `__this__`, resolve to the `this` constant and halt.
 13. Generate a local variable named `i`, resolve to it and halt.
-14. Fail
+14. Fail.
```

Comment:
Hmm, really? This isn't exactly a sentence...

Reply:
It is. Unlike the sentence fragments on lines 540 onward :)

Reply:
I'm unused to editing lists (not what I typically handle honestly) and made a style decision to preface every list with a colon before it begins and end every item with a period. I can revert these changes if you want.